web/styles/drafts.css

```diff
@@ -1,6 +1,10 @@
 .drafts-container {
     .banner-container {
         margin: 0 25px 5px;
+
+        .banner-label {
+            text-align: left;
+        }
```

Comment:
Okay, seems like if we aren't defining `text-align: left` over here, the `banner-label` is inheriting the `text-align: center` value from the `.overlay-messages-header` class. I expect we will run across instances where the parent element has a different text alignment than what we desire. Since the navbar banners explicitly have a `text-align: center` applied on them, we should just apply the `text-align: left` property to the `.banner-label` class in `web/styles/banners.css`.

Reply:
I just wanted to have this change be specific for drafts UI for now to avoid introducing any unintentional bugs. So, maybe we can do that as a follow-up.

Reply:
Sure thing, we can go ahead and do this for all banners as a follow-up to allow this PR to be bug free.